R/utils-validation.R

```diff
@@ -84,12 +84,13 @@ accepted_parameter_value <- list(smoothing_method = c("LOESS"),
 #TODO reconsider if can return FALSE
 #' Check if valid distribution list
 #'
-#' Throws an error if not a list, or not a list with the appropriate elements.
-#' Returns FALSE if parameter values return an improper distribution (if gamma distr)
+#'
 #'
 #' @inheritParams distribution
+#' @inheritParams validation_utility_params
+#'
 #'
-#' @return a boolean value.
+#' @return boolean. Returns FALSE if parameter values return an improper distribution (if gamma distr). Throws an  Throws an error if not a list, or not a list with the appropriate elements. Returns TRUE otherwise.
```

Comment:
```suggestion
#' @return boolean. Returns FALSE if parameter values return an improper distribution (if gamma distr). Throws an error if not a list, or not a list with the appropriate elements. Returns TRUE otherwise.
```